assets/bwats-release/jobs/check-system/templates/run.ps1.erb

```diff
@@ -264,28 +264,18 @@ function Verify-InstalledFeatures {
       Write-Host "Found $feature feature"
     }
   }
-  function Assert-DefenderFeaturesDisabled {
-    Get-MpPreference |
-    Select -ExpandProperty CimInstanceProperties |
-    Where-Object { $_.Name -Like "Disable*" } |
-    ForEach-Object {
-      if ($_.Value -eq $false) {
-        if ($_.Name -eq "DisableFtpParsing") {
-          Write-Host "**************************"
-          Write-Host "Skipping $($_.Name); it's not settable. It was $($_.Value)"
-          Write-Host "<https://www.pivotaltracker.com/story/show/180628580/comments/228655194>"
-          Write-Host "**************************"
-        }
-        else {
-          Write-Error "Expected Windows Defender property $($_.Name) to be True, because all Defender features should be disabled, it was false"
-          Exit 1
-        }
-      }
-      else {
-        Write-Host "$($_.Name) property is correctly set to $($_.Value)"
-      }
+  function Assert-IsNotInstalled {
+    param (
+      [string] $feature= (Throw "feature param required")
+    )
+    If (!(Get-WindowsFeature $feature).Installed) {
+      Write-Error "Feature $feature is not installed"
```

Comment:
Seems like `Write-Error` should be used in the error case, and `Write-Host` otherwise as in `Assert-IsInstalled`

Reply:
Good catch!